src/app/compliance/actions.js

```diff
@@ -199,3 +200,43 @@ export const removeInvestors = (addresses: Array<Address>) => async (dispatch: F
     true // TODO @bshevchenko
   ))
 }
+
+export const getFreezeStatus = () => async (dispatch: Function, getState: GetState) =>{
+  // $FlowFixMe
+  getState().token.token.contract.subscribe('LogFreezeTransfers', {}, (e)=>{
+    if(e.returnValues['0']){
```

Comment:
can be simplified into `dispatch({ type: FREEZE_STATUS, freezeStatus: event.returnValues._freeze })`
better to avoid using `e` as variable name since it's commonly used for errors